Extensions/P2P/B_p2ptools.ts

```diff
@@ -168,25 +231,9 @@ namespace gdjs {
         eventName: string,
         defaultDataLoss: boolean
       ): boolean => {
-        const dataLoss = eventHandling[eventName];
-        if (dataLoss == undefined) {
-          eventHandling[eventName] = defaultDataLoss;
-          return onEvent(eventName, defaultDataLoss);
-        }
-        if (dataLoss) {
-          let returnValue = triggeredEvents[eventName];
-          if (typeof returnValue === 'undefined') {
-            return false;
-          }
-          triggeredEvents[eventName] = false;
-          return returnValue;
-        } else {
-          let returnValue = lastEventData[eventName];
-          if (typeof returnValue === 'undefined') {
-            return false;
-          }
-          return returnValue.length !== 0;
-        }
+        const event = getEvent(eventName);
+        event.dataloss = defaultDataLoss;
```

Comment:
I know that this was already implemented like this, but should have this "dataloss" assignment have been done in a separate "setEventDataloss" set up function (which would have been exposed as a set up action)? 
I guess the idea was to surface this choice directly in the condition?

Reply:
> I guess the idea was to surface this choice directly in the condition?

Yes, I am afraid making it a separate event might make it a "scary advanced feature", but it is essential for things like position updates.